public/r2/v1/swagger/swagger.json

```diff
@@ -877,6 +943,66 @@
                 }
             }
         },
+        "RuleSetChange": {
+            "type": "object",
+            "properties": {
+                "operation": {
+                    "type": "string",
+                    "enum": ["add", "remove", "change"],
```

Comment:
Does this mean we might add / delete namespace as part of the bulk operation? Might wanna think about if we want to allow that given that we already have APIs for adding new namespaces and deleting existing namespaces?

Reply:
This was something I was thinking about. It does seem like this endpoint would be more simple if we did not allow for these types of operations, but if users are going to be hitting this endpoint programmatically it might be easier to have a one-stop endpoint for manipulating rules. I can remove it for now and if we need to add it back later we can.